internal/chartverifier/verifier.go

```diff
@@ -115,13 +118,23 @@ func (c *verifier) Verify(uri string) (*apiReport.Report, error) {
 			ViperConfig:      c.subConfig(string(check.CheckId.Name)),
 			AnnotationHolder: &holder,
 			Timeout:          c.timeout,
+			PublicKeys:       c.publicKeys,
 		})
 
 		if checkErr != nil {
 			return nil, NewCheckErr(checkErr)
 		}
 		_ = result.AddCheck(check, r)
 
+		if check.CheckId.Name == apiChecks.SignatureIsValid {
+			if len(c.publicKeys) == 1 && strings.Contains(r.Reason, checks.ChartSigned) {
```

Comment:
The public keys is an array of string, but here we are checking whether it contains only one public key in c.publicKeys array. So if the user has to specify the public key per chart then why not have the type of c.publicKeys as a string instead of array.

Reply:
With an array we do not know which key was the correct one so cannot add digest to report. With cli we only get one key so if chart verifies we know that is the correct key so we can then add digest to report.